src/main/java/org/broadinstitute/hellbender/tools/copynumber/coverage/segmentation/CopyRatioSegment.java

```diff
@@ -0,0 +1,96 @@
+package org.broadinstitute.hellbender.tools.copynumber.coverage.segmentation;
+
+import htsjdk.samtools.util.Locatable;
+import org.broadinstitute.hellbender.utils.GATKProtectedMathUtils;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.param.ParamUtils;
+
+import java.util.List;
+
+public class CopyRatioSegment implements Locatable {
+    private final SimpleInterval interval;
+    private final int numPoints;
+    private final double meanLog2CopyRatio;
+
+    public CopyRatioSegment(final SimpleInterval interval,
+                            final int numPoints,
+                            final double meanLog2CopyRatio) {
+        Utils.nonNull(interval);
+        ParamUtils.isPositiveOrZero(numPoints, "Number of copy-ratio points must be non-negative.");
+        this.interval = interval;
+        this.numPoints = numPoints;
+        this.meanLog2CopyRatio = meanLog2CopyRatio;
+    }
+
+    public CopyRatioSegment(final SimpleInterval interval,
+                            final List<Double> denoisedLog2CopyRatios) {
+        Utils.nonNull(interval);
+        Utils.nonNull(denoisedLog2CopyRatios);
+        this.interval = interval;
+        numPoints = denoisedLog2CopyRatios.size();
+        final double meanCopyRatio = denoisedLog2CopyRatios.stream().mapToDouble(log2CR -> Math.pow(2., log2CR)).average().orElse(Double.NaN);
+        meanLog2CopyRatio = Math.log(meanCopyRatio) * GATKProtectedMathUtils.INV_LOG_2;
+    }
+
+    @Override
+    public String getContig() {
+        return interval.getContig();
```

Comment:
I'm in favor of putting trivial getters and setters on a single line, but it's your code.